force_wfmanager/ui/setup/communicator/communicator_view.py

```diff
@@ -0,0 +1,65 @@
+from traits.api import (
+    Instance, List, on_trait_change, Event, HasTraits
+)
+from traitsui.api import View, Item
+
+from force_bdss.api import Workflow
+from .notification_listener_view import NotificationListenerView
+
+
+class CommunicatorView(HasTraits):
```

Comment:
This is a new class designed to contain the list of `NotificationListener`s in the `Workflow`. At the moment it doesn't add anything to the design, except equalising the hierarchy of objects in the `WorkflowView`: the `MCOView`, `ProcessView` and CommunicatorView` classes all have level 2 status and contain lists of views.